src/main/java/org/sunbird/user/service/UserUtilityService.java

```diff
@@ -49,4 +50,5 @@ public void getUserDetailsFromDB(List<String> userIds, List<String> fields,
 	Map<String, Map<String, Object>> getUserDetailsFromES(List<String> userIdList, List<String> userFields) throws IOException;
 
 	Map<String, Object> getUsersDataFromLookup(String email, String authToken);
+	SBApiResponse getRecommendation(String authUserToken, Map<String, Object> orgRequest);
```

Comment:
use proper method name... we are not trying to get something...

Reply:
updated